endpointmanager/pkg/endpointmanager/postgresql/healthitproductstore.go

```diff
@@ -148,11 +148,6 @@ func (s *Store) GetHealthITProductUsingNameAndVersion(ctx context.Context, name
 // GetHealthITProductsUsingVendor returns a slice of HealthITProducts that were created by the given vendor_id
 func (s *Store) GetHealthITProductsUsingVendor(ctx context.Context, vendorID int) ([]*endpointmanager.HealthITProduct, error) {
```

Comment:
It looks like GetHealthITProductsUsingVendor is not being used anywhere anymore so we can delete it unless you see a need in the near future?